medicines/pars-upload/src/products_form.js

```diff
@@ -83,6 +83,12 @@ export const Products = ({
     }
   }
 
+  const onAddAnotherSubstance = (event) => {
+    event.preventDefault()
+    setSubstanceIds((ids) => [...ids, getNextId()])
+    onAddAnotherSubstance
```

Comment:
Is this right, or am I missing something?

Reply:
Nope - it's not. Good spot!